README.rst

```diff
@@ -2,93 +2,103 @@
 ``sunpy``
 *********
 
-|Latest Version| |DOI| |repostatus| |python|
-|ci| |codecov| |Docs|
+|Latest PyPi Version|
+|Latest Conda Version|
```

Comment:
Annoyingly this shows the latest uploaded version (currently 5.0.2) and not the highest version number (currently 5.1.0). Not sure there's an easy way to fix that, but thought I'd flag.

Reply:
Yeah, I plan to open an issue upstream about this.

Reply:
https://github.com/ContinuumIO/anaconda-issues/issues/13320